### ISSUE TITLE
Add others Pokemon features

### ISSUE THREAD

**Issue by ghost** [Tuxemon/Tuxemon#364]
I think it would be nice to add features from other Pokemon games in the not too distant future.

Many players or former Pokemon players miss some things from those games for example, I miss the basement of Pokemon Platinum and even the previous games contests of the Game Boy, the secret bases and the connections of other players to your world through the Nexus Zone of Pokemon Black and White.

**Comment by xirsoi** [comment]
A few of the developers (myself included) have only played the early generations and at least one never played Pokemon at all. Could you describe the features you'd like to see in some detail?

It would be helpful if you said what it was about those features that you enjoyed. How did they make you feel?

**Comment by CitoplasmaX** [comment]
https://bulbapedia.bulbagarden.net/wiki/The_Underground 

https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9mon_Contest

https://bulbapedia.bulbagarden.net/wiki/Pok%C3%A9athlon_Dome

**Comment by CitoplasmaX** [comment]
Using those characteristics, already at first sight they made me feel the immensity of the Pokemon world and the long hours of fun that would happen, above all, among friends, stealing flags, hiding objects in the worlds of another, exploring unexplored areas of those places, discovering their secrets, the laughter that we used when we lost in the contests for some nonsense or trying to make a Pokemon stand out and an almost infinite world of possibilities.

Basically it made me feel more attached to that world, at least some of those people who played with me share that feeling and I hope that other people too, is something that I would love to see in this game in some future.

**Comment by xirsoi** [comment]
The Underground concept sounds very interesting. I can see something like that being implemented by content designers (with support from developers, i'm sure). The other features also seem largely doable through scripting and map creation. Although, dev work will probably be required to add some scripting functionality.

The multiplayer aspects, however will likely require a lot of work.

**Comment by Qiangong2** [comment]
Right now, I believe it is more important to have a fully functioning story/game before adding the edits you suggest. This is a good idea, but I think we should get the fundamentals done before working on this.

**Comment by xirsoi** [comment]
I agree, never the less, thank you for your suggestions @CitoplasmaX !

**Comment by CitoplasmaX** [comment]
I understand how difficult it must be to add all these characteristics, that is why I propose them for a future.

One thing that reminded me of Xirsoi when he named the creation of the map was just an idea that I liked that was once implemented in Pokemon but that, given its commercial origin, did not interest. It consisted in that the subsequent editions of the saga, in this case of this game, added extensions to the map with the new regions, thus being a true Pokemon adventure of a trainer throughout the Pokemon world and not doing reboot in each beginning, besides, give the possibility to players to start in a certain region or at random.

**Comment by bitcraft** [comment]
I've never played any pokemon, unless you count Smash Bros.  ^_^  If you have any specific ideas, the forums are a good place to discuss potential ideas, whereas GitHub Issues are a bit better for technical aspects of actually implementing them.  

Multiplayer was working pretty well in the past, and it is still an objective to have a client/server model for Tuxemon.  A lot of the work that I do has this goal in mind.

**Comment by CitoplasmaX** [comment]
Please, read this. https://guides.github.com/features/issues/

It is not only for technical aspects.

**Comment by bitcraft** [comment]
@CitoplasmaX The forums have more participation and information related to the story and gameplay features.  I've been a part of this project for a few years now, I'm just letting you know how the community works.  Github issues eventually get closed and hidden, forum threads stay open all the time.

https://forum.tuxemon.org/

**Comment by CitoplasmaX** [comment]
Quiet, alone alone was blinded in the use of the themes of GitHub, is that there are already three people who have thrown me the same and in the other previous occasions I have closed the problem by making proposals, then I got fed up and since I'm not sure if you also thought so I preferred to release it. I appreciate the invitation, as soon as I can translate the ideas.
